tests/io/dask/dataframe/test_repartition.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+import pytest
+
+from kartothek.core.factory import DatasetFactory  # noqa: E402
+from kartothek.io.dask.dataframe import read_dataset_as_ddf
+from kartothek.io.dask.repartition import (
+    calculate_repartition_ratio,
+    optimize,
+    repartition_ddf,
+)
+from kartothek.io.iter import store_dataframes_as_dataset__iter
+from kartothek.io_components.metapartition import SINGLE_TABLE
+
+
+@pytest.fixture
+def dataset_factory_function_scope(store_factory):
+    return DatasetFactory(dataset_uuid="test", store_factory=store_factory)
```

Comment:
I would prefer defining these in the test functions instead of using a fixture. The fixture itself is not functional since the factory only works for reading a dataset but the fixture doesn't set up the dataset. Therefore this factory is non-functional and the value of defining it in the fixture is marginal.

Reply:
Agreed. Removed this fixture and its usage. Using store_factory wherever required. Please check and confirm.